crates/ruff/src/rules/pyflakes/rules/imports.rs

```diff
@@ -13,6 +13,7 @@ use crate::checkers::ast::Checker;
 pub struct UnusedImport {
     pub name: String,
     pub ignore_init: bool,
+    pub in_except_handler: bool,
```

Comment:
Not sure if I'm blind but where do we initialize `in_except_handler`? 

Is it possible for both `ignore_init` and `in_except_handler` to be true? If not, how about we create an enum instead `UnusedImportKind`